jumble/src/components/commands.ts

```diff
@@ -447,9 +447,6 @@ async function handleUseDataInSpell(ctx: CommandContext) {
           const charm = await ctx.charmManager.get(ctx.focusedCharmId);
           if (!charm) throw new Error("No current charm found");
           const argument = ctx.charmManager.getArgument(charm);
-          if (!argument) {
-            throw new Error("No sourceCell/argument found for current charm");
-          }
```

Comment:
I don't see why this needed to go...

Reply:
<img width="829" alt="image" src="https://github.com/user-attachments/assets/899f544a-6aed-4b0e-9c09-e147d6241b2d" />

because now we throw instead of returning undefined